Dockerfile

```diff
@@ -0,0 +1,60 @@
+# See: https://github.com/vercel/next.js/blob/canary/examples/with-docker/
+
+# Install dependencies only when needed
+FROM node:16-alpine AS deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies based on the preferred package manager
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i; \
+  else echo "Lockfile not found." && exit 1; \
+  fi
+
+
+# Rebuild the source code only when needed
+FROM node:16-alpine AS builder
+WORKDIR /app
+ARG APP_DEPLOYMENT=INCIDENT_MANAGER
+
+COPY --from=deps /app/node_modules ./node_modules
+COPY . .
+
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+ENV NEXT_TELEMETRY_DISABLED 1
+
+ENV NEXT_PUBLIC_APP_DEPLOYMENT=${APP_DEPLOYMENT}
+RUN NEXT_STANDALONE_DEPLOYMENT=true npm run build
+
+# Production image, copy all the files and run next
+FROM node:16-alpine AS runner
+WORKDIR /app
+
+ENV ORY_KRATOS_URL=${ORY_KRATOS_URL}
```

Comment:
I don't think this can be part of the dockerfile

Reply:
Didn't get that. It's dynamic environment variable. Needed for the application to figure out the kratos backend.

Reply:
It can be set at the runtime. Or picks up value from the time it was built.

Reply:
Got it. Will remove from here.